docs/pages/database-access/guides/snowflake.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Database Access with Snowflake
+title: Database Access with Snowflake (Preview)
```

Comment:
If we adjusted the title in `config.json` for the other page, shouldn't we do the same for this one?

Reply:
@alexfornuto it does in `config.json`, just not in the page

Reply:
So it does... but the title there doesn't mention Redis. Not sure if it should or not, just saying.

Feel free to resolve this thread if the sidebar title shouldn't include Redis.